compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -2288,7 +2288,13 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             }
 
             let field_kind = if is_accessible { "field" } else { "private field" };
-            err.span_label(item_name.span, format!("{field_kind}, not a method"));
+            err.span_label(
+                item_name.span,
+                format!(
+                    "`{item_name}` is a {field_kind} \
```

Comment:
Repeating `item_name` here is redundant -- we're already using a label to point to the item name itself, and `item_name` can be arbitrarily long.